api-test/src/main/resources/injiverify/VpRequestForSessionRequest/VpRequestForSessionRequest.yml

```
@@ -0,0 +1,232 @@
+            }
+          }
+        ]
+      }
+    }'
+    output: '{
+      "status": "400"
+}'
+  Injiverify_CreateNewVerificationRequest_ForVpSessionRequest_withResponseCodeValidationRequired_asFalse_Valid_Sid:
+    endPoint: /v1/verify/vp-session-request
+    description: Creates a VP request with valid data and responseCodeValidationRequired is false for VP Session result API
+    description: Creates a VP request with valid data and responseCodeValidationRequired is false for VP Session result API
```

YAMLlint (1.38.0)

[error] duplication of key "description" in mapping

(key-duplicates)